File: csharp/VS2008/netframework/Modules/20.Reports/88.Generic Reports/EnterSQLDialog.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;

namespace GenericReports
{
    /// <summary>
    /// A dialog where you can enter any SQL.
    /// </summary>
    public partial class EnterSQLDialog: System.Windows.Forms.Form
    {

        public EnterSQLDialog()
        {
            InitializeComponent();
        }

        public string SQL
        {
            get
            {
                return edSQL.Text;
            }
        }
    }
}

File: vb/VS2012/netframework/Modules/20.Reports/88.Generic Reports/EnterSQLDialog.vb
Imports System.Collections
Imports System.ComponentModel

Namespace GenericReports
	''' <summary>
	''' A dialog where you can enter any SQL.
	''' </summary>
	Partial Public Class EnterSQLDialog
		Inherits System.Windows.Forms.Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Public ReadOnly Property SQL() As String
			Get
				Return edSQL.Text
			End Get
		End Property
	End Class
End Namespace
